runelite-client/src/main/java/net/runelite/client/plugins/droplogger/data/LoggedItem.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2018, Woox <https://github.com/wooxsolo>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.droplogger.data;
+
+import java.awt.image.BufferedImage;
+import lombok.Getter;
+import lombok.Setter;
+import net.runelite.api.ItemComposition;
+
+public class LoggedItem
+{
+	@Getter
+	private int itemId;
+
+	@Getter
+	private int quantity;
+
+	@Getter
+	private ItemComposition composition;
+
+	@Getter
+	@Setter
+	private Integer price;
+
+	@Getter
+	@Setter
+	private BufferedImage image;
+
+	public LoggedItem(int itemId, int quantity, ItemComposition composition, Integer price)
+	{
+		this.itemId = itemId;
+		this.quantity = quantity;
+		this.composition = composition;
+		this.price = price;
+	}
+
+	public long getValue()
+	{
+		if (price != null)
+		{
+			return (long)price * quantity;
+		}
+		if (composition != null)
+		{
+			return (long)(composition.getPrice() * 0.6 * quantity);
+		}
+		return 0;
```

Comment:
Add line above

Reply:
I believe a return statement is an exception to the blank line after closing brace rule

Reply:
It is not